### PR TITLE
fix log format

### PR DESCRIPTION
**Pull request by yaohcn**

### CODE REVIEW

File: chain/events/events.go [hunk 1]
```diff
@@ -129,7 +129,7 @@ func (e *Events) listenHeadChangesOnce(ctx context.Context) error {
 	}
 
 	if err := e.tsc.add(cur[0].Val); err != nil {
-		log.Warn("tsc.add: adding current tipset failed: %w", err)
+		log.Warnf("tsc.add: adding current tipset failed: %w", err)
```
Comment (Kubuxu) [inline chain/events/events.go:132]: Can you also fix the `%w` format here, it should be `%v`.

**Review by Kubuxu — APPROVED** [review]
SGWM

File: chain/events/events.go (hunk 1 above)
Comment (magik6k) [inline chain/events/events.go:132]:
```suggestion
		log.Warnf("tsc.add: adding current tipset failed: %v", err)
```

**Review by magik6k — APPROVED** [review]
Thanks!

**Pull request merged**